automatic/clipboardfusion/clipboardfusion.nuspec

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>clipboardfusion</id>
+    <version>0.0</version>
+    <title>ClipboardFusion</title>
+    <authors>Binary Fortress Software</authors>
+    <licenseUrl>https://www.clipboardfusion.com/License/</licenseUrl>
+    <projectUrl>https://www.clipboardfusion.com</projectUrl>
+    <iconUrl>https://images.binaryfortress.com/ChocolateyIcons/ClipboardFusion.png</iconUrl>
+    <docsUrl>https://www.clipboardfusion.com/FAQ/</docsUrl>
+    <mailingListUrl>https://www.clipboardfusion.com/Discussions/</mailingListUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+    ClipboardFusion makes it easy to remove clipboard text formatting,
+    replace clipboard text or run powerful macros on your clipboard contents.
+    You can even sync your clipboard with other computers and mobile devices.
```

Comment:
Add a notes heading as per [§.1.2.3](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#123-informative-description) with a paragraph mentioning that the free license is only for personal & non-commercial use